.github/workflows/test.yaml

```diff
@@ -21,13 +21,14 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: "1.19"
+          cache: true
       - uses: debianmaster/actions-k3s@v1.0.5
         with:
           version: 'v1.23.6-k3s1'
       - run: make validate-ci
       - run: make validate
       - run: make build
       - run: docker buildx install
-      - run: make setup-ci-image
+      - run: make setup-ci-image -j2
```

Comment:
curious why you added -j to just these two specific steps. And why you chose 2

Reply:
Removed from this step, turns out it wasn't necessary. Ran with -j2 later on because there are two makefile dependencies in make test, so it would run each of those in a separate thread.